regression/cbmc-sequentialization/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
+  set(gcc_only "-X gcc-only")
```

Comment:
No quotes, this is the same bug fotis was struggling on windows with :)

Reply:
Should be fixed now.

Reply:
Actually none of this is going to work on Windows as-is, because pthreads isn't natively supported on that platform. I'll either need to disable all these tests, or add a windows test-time dependency for a suitable compatibility layer.